style.css

```diff
@@ -0,0 +1,277 @@
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1.85rem;
+    margin-bottom: 1rem;
+    text-transform: capitalize;
+    color: #fffbfc;
+  }
+  .hero-info p {
+    font-size: 1.08rem;
+    line-height: 1.2rem;
+    color: rgb(241, 234, 234);
+  }
+  /* ===Social links== */
+  .social-media {
+    gap: 0.5rem;
+    position: absolute;
+    bottom: 2rem;
+    right: 4rem;
+  }
+  .social-media a {
+    height: 28px;
+    width: 28px;
+    background-color: #ee1e33;
+    font-size: 1.2rem;
+    color: rgb(255, 255, 255);
+    text-align: center;
+    line-height: 25px;
+    border-radius: 20%;
+    transition: all 0.5s ease-in-out;
+  }
+  .social-media a:hover {
+    background-color: #c5606a;
+    transform: scale(0.98);
+  }
+  @media (min-width: 514px) {
+    .hero-info {
+      width: 50ch;
+    }
+  }
+  @media (min-width: 700px) {
+    .hero-info p {
+      font-size: 1.2rem;
+      width: 50ch;
+    }
+    .hero-info h1 {
+      font-size: 2rem;
+    }
+  }
+  @media (min-width: 967px) {
+    .hero-info h1 {
+      font-size: 2.2rem;
+      line-height: 1.9rem;
+    }
+    .social-media a {
+      width: 40px;
+      height: 40px;
+      line-height: 40px;
+    }
+  }
+  
+  /* ===Blog=== */
+  main {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+  .blog.grid {
+    gap: 2rem;
+    max-width: 967px;
+  }
+  .blog-img {
+    display: grid;
+    justify-items: center;
+  }
+  .blog img {
+    width: 350px;
+    height: 300px;
+  }
+  .blog-info {
+    display: grid;
+    justify-items: center;
+    align-content: center;
+    margin: 0;
+  }
+  .blog h3 {
+    font-family: Arial, Helvetica, sans-serif;
+    color: #ff6978;
+    text-shadow: 1px 1px 2px rgb(151, 146, 146);
+    letter-spacing: 0.1rem;
+  }
+  .blog p {
+    width: 50ch;
+    letter-spacing: 0.05rem;
+    line-height: 1.2rem;
+  }
+  .spec i {
+    color: white;
+    font-weight: bold;
+    background-color: rgb(0, 0, 0);
+    font-size: 13px;
+    width: 15px;
+    line-height: 15px;
+    height: 15px;
+    border-radius: 50%;
+    text-align: center;
+    margin-right: 1rem;
+  }
+  @media (min-width: 850px) {
+    .blog-info {
+      max-width: 850px;
+      justify-items: flex-start;
+    }
+    .blog {
+      grid-template-columns: 1fr 2fr;
+      justify-items: center;
+    }
+    .blog p {
+      width: 50ch;
+    }
+    .blog img {
+      align-self: flex-end;
+    }
+  }
+  
+  /* ===Newslletter=== */
+  
+  .newsletter {
+    padding: 1.5em;
+    background-color: black;
+    margin-inline: auto;
+    width: 100%;
+  }
+  .input {
+    width: 100%;
+  }
+  .form-bg {
+    background-color: white;
+    width: 100%;
+    display: flex;
+    padding: 0.51em;
+    max-width: 600px;
+    margin-inline: auto;
+    justify-content: space-between;
+  }
+  
+  /* ===footer=== */
+  .copy {
+    background-color: rgba(0, 0, 0, 0.897);
+    padding-block: 0.5em;
+    margin-top: 3rem;
+  }
+  .col {
+    color: #ee1e33;
+  }
+  
```

Comment:
There is no proper spacing on here.

As good code practice, there ought to be a new line after every block
e.g '}'...newline before some new selector

Reply:
:heavy_check_mark: